tox.ini

```diff
@@ -52,6 +52,8 @@ deps =
      certifi
      # VCS hooks support
      py36,!coverage: pre-commit
+     # Does our OpenMetrics endpoint adhere to the spec:
+     prometheus-client==0.11.0
```

Comment:
The Nix packaging will need this dependency added, too (see `checkInputs` in `nix/tahoe-lafs.nix` - I think `prometheus_client` is the name in nixpkgs, though I'm not sure what version is there).

Reply:
It's 0.9.0 in 21.01, so I guess it's way-too-old in 19.09 which is used for the tests IIANM. I'll just push it and see what CI says.